packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -844,6 +844,139 @@ integTest('cdk ls', withDefaultFixture(async (fixture) => {
   }
 }));
 
+/**
+ * Type to store stack dependencies recursively
+ */
+export type DependencyDetails = {
```

Comment:
Do we need to export these?

Reply:
Not needed to export. Nice catch, will update!